### PR TITLE
Global search usability improvements

### PR DESCRIPTION
**Pull request by matthiasgeiger**
Some minor changes:
- removed relict in `JabRefFrame.java`
- reenabled "Search globally" Shortcut (if not activated global search is activated and cursor jumps to search field)
- Changed tooltip to "Show global search results in window" to be consistent with "non-global search"

Will merge if there are no objections until this evening ;-)

- ~~[ ] Change in CHANGELOG.md described~~ still unreleased
- ~~[ ] Tests created for changes~~
- ~~[ ] Screenshots added (for bigger UI changes)~~
- [x] Manually tested changed features in running JabRef
- [x] documentation status: https://github.com/JabRef/help.jabref.org/pull/128
- [x] If you changed the localization: Did you run `gradle localizationUpdate`?

### CODE REVIEW

**Comment by koppor** [comment]
@bartsch-dev can you have a look?

File: src/main/java/net/sf/jabref/gui/search/GlobalSearchBar.java [hunk 1]
```diff
@@ -85,18 +86,33 @@ public GlobalSearchBar(JabRefFrame frame) {
...
-        globalSearch.addActionListener(e -> {
-            searchPreferences.setGlobalSearch(globalSearch.isSelected());
-            if (globalSearch.isSelected()) {
-                openCurrentResultsInDialog.setToolTipText(Localization.lang("Search in all open databases"));
-            } else {
-                openCurrentResultsInDialog.setToolTipText(Localization.lang("Show search results in a window"));
+        globalSearch.setToolTipText(Localization.lang("Search in all open databases"));
+
+
+        AbstractAction clickAction = new AbstractAction() {
```
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/search/GlobalSearchBar.java]: Please rename.

File: src/main/java/net/sf/jabref/gui/search/GlobalSearchBar.java [hunk 2]
```diff
@@ -85,18 +86,33 @@ public GlobalSearchBar(JabRefFrame frame) {
...
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                searchPreferences.setGlobalSearch(globalSearch.isSelected());
+                updateOpenCurrentResultsTooltip(globalSearch.isSelected());
             }
-        });
+        };
+
+        AbstractAction shortCutAction = new AbstractAction() {
```
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/search/GlobalSearchBar.java]: Please rename.

**Comment by matthiasgeiger** [comment]
@bartsch-dev I wanted to be consistent with the CTRL-F shortcut which also moves the cursor to the search field

**Pull request merged**